scale-tests/README.md

```diff
@@ -0,0 +1,34 @@
+# Spark Scale Tests
+
+## Batch
+
+## Streaming
+To run the Kafka-Spark Streaming word count app:
+- Install kafka
+  ```bash
+  dcos package install kafka --yes
+  ```
+- Deploy spark dispatchers
+  ```bash
+  Usage: deploy-dispatchers.py [num_dispatchers] [service_name_base] [output_file] (options_json_file)
```

Comment:
This changed slightly since Scott's latest commit: https://github.com/mesosphere/spark-build/commit/2ac5e46822120562c247670fb362066801a41854